openshift/templates/images/nodejs/Dockerfile

```diff
@@ -49,5 +49,5 @@ EXPOSE 3000
 # can execute relative to this path.
 WORKDIR ${HOME}
 
-# Set the default CMD for the image
+# Set the default CMD for the image, without it will fail as no script usage exists.
```

Comment:
i don't understand what the 'usage' command is

Reply:
its fake, if you are using this image, and don't specify a command it will fail... i'm just sorting some of this stuff out now, will show you once i have it sorted.